boards/nxp/frdm_mcxw72/doc/index.rst

```diff
@@ -0,0 +1,158 @@
+.. zephyr:board:: frdm_mcxw72
+
+Overview
+********
+
+The FRDM-MCXW72
+
+The MCX W72x family features a 96 MHz Arm® Cortex®-M33 core coupled with a
+multiprotocol radio subsystem supporting Matter, Thread, Zigbee and
+Bluetooth LE. The independent radio subsystem, with a dedicated core and
+memory, offloads the main CPU, preserving it for the primary application and
+allowing firmware updates to support future wireless standards.
+
+Hardware
+********
+
+- MCXW72 Arm Cortex-M33 microcontroller running up to 96 MHz
+- 2MB on-chip Flash memory unit
+- 256 KB TCM RAM
+- On-board MCU-Link debugger with CMSIS-DAP
+
+For more information about the MCXW72 SoC and FRDM-MCXW72 board, see:
+
+- `MCXW72 SoC Website`_
+- `FRDM-MCXW72 Website`_
+
+Supported Features
+==================
+
+The ``frdm_mcxw72`` board target in Zephyr currently supports the following features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinctrl                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| LPUART    | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| TPM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| LPTMR     | on-chip    | counter                             |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
```

Comment:
The other rows put a specific peripheral name, like LPUART, this is LPSPI right

Reply:
Updated.